frontend/src/Frontend/Wallet.hs

```diff
@@ -308,6 +321,33 @@ storeKeys ks = setItemStorage localStorage StoreWallet_Keys ks
 loadKeys :: (FromJSON key, HasStorage m, MonadJSM m) => m (Maybe (Accounts key))
 loadKeys = getItemStorage localStorage StoreWallet_Keys
 
+uiAccountNameInput
```

Comment:
This appears to be a UI thing in Frontend.Wallet Probs should go in Frontend.UI.*